src/styles/blocks/card.scss

```diff
@@ -0,0 +1,132 @@
+.catalog__card {
+  padding-top: 32px;
+  margin-bottom: 48px;
+  margin-right: 48px;
+  width: 200px;
+  height: 406px;
+}
+
+.card {
+  @include flex();
+  justify-content: center;
+  align-content: flex-start;
+  font-family: inherit;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
```

Comment:
it is a good practice to add `box-sizing: border-box;` on all elements using `*`